assets/js/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+// eslint-disable-next-line implicit-dependencies/no-implicit
+require('hof/frontend/themes/gov-uk/client-js');
+
+var $ = require('jquery');
+
+var accessibleAutocomplete = require('accessible-autocomplete');
+$('.typeahead').each(function applyTypeahead() {
+  accessibleAutocomplete.enhanceSelectElement({
+    defaultValue: '',
+    selectElement: this
+  });
+})
```

Comment:
if you're not using, I wouldn't add it.  It makes sense to add it as part of a PR if I'm honest.

Reply:
I have removed this now